pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -215,7 +215,9 @@ func (p *staticPolicy) validateState(s state.State) error {
 	// 1. Check if the reserved cpuset is not part of default cpuset because:
 	// - kube/system reserved have changed (increased) - may lead to some containers not being able to start
 	// - user tampered with file
-	if !p.reservedCPUs.Intersection(tmpDefaultCPUset).Equals(p.reservedCPUs) {
+	// 2. This only applies when managed mode is disabled. Active workload partitioning feature
+	//    removes the reserved cpus from the default cpu mask on purpose.
+	if !managed.IsEnabled() && !p.reservedCPUs.Intersection(tmpDefaultCPUset).Equals(p.reservedCPUs) {
```

Comment:
is it ok to skip this check entirely when in managed mode? Or do we have an equivalent check?

Reply:
Yes, the check is not valid for the managed mode. The extra check for not using reserved is lower in the file (totalKnownCPUs == availableCPUs)